doc/lightning-listtransactions.7.md

```diff
@@ -0,0 +1,100 @@
+lightning-listtransactions -- Command to get the list of transactions that was stored in the wallet
+============================================================
+
+SYNOPSIS
+--------
+
+**listtransactions**
+
+DESCRIPTION
+-----------
+
+The **listtransactions** command returns transactions tracked in the wallet. This includes deposits, withdrawals and transactions related to channels. A transaction may have multiple types, e.g., a transaction may both be a close and a deposit if it closes the channel and returns funds to the wallet.
+
+EXAMPLE JSON REQUEST
+------------
+```json
+{
+  "id": 82,
+  "method": "listtransactions",
+  "params": {}
+}
+```
+
+RETURN VALUE
+------------
+
+On success, the command will return a list of transactions, each object rappresent the transaction with all details.
```

Comment:
s/rappresent the transaction with all details./represents a transaction, with the following details:/